SampleGallery/Samples/SDK 17134/RadialGradients/RadialGradients.xaml.cs

```diff
@@ -0,0 +1,310 @@
+
+                _changeButtonGradientStop2 = _compositor.CreateColorKeyFrameAnimation();
+                _changeButtonGradientStop2.InsertKeyFrame(0, _orangeRed);
+                _changeButtonGradientStop2.InsertKeyFrame(1, _lightBlue);
+                _changeButtonGradientStop2.Duration = TimeSpan.FromSeconds(2);
+
+                _BBGradientStop2.StartAnimation(nameof(_BBGradientStop2.Color), _changeButtonGradientStop2);
+
+                // Animations for the pulsing effect are turned off
+                _stop1OffsetAnim = _compositor.CreateScalarKeyFrameAnimation();
+                _stop1OffsetAnim.InsertKeyFrame(0, 0);
+                _stop1OffsetAnim.InsertKeyFrame(0f, 0f);
+                _stop1OffsetAnim.Duration = TimeSpan.FromSeconds(1.5f);
+                _stop1OffsetAnim.IterationCount = 50;
+                
+                _PBGradientStop1.StartAnimation("Offset", _stop1OffsetAnim);
+
+                _stop2offsetAnim2 = _compositor.CreateScalarKeyFrameAnimation();
+                _stop2offsetAnim2.InsertKeyFrame(0, 0);
+                _stop2offsetAnim2.InsertKeyFrame(0f, 0f);
+                _stop2offsetAnim2.Duration = TimeSpan.FromSeconds(1.5f);
+                _stop2offsetAnim2.IterationCount = 50;
+                _stop2offsetAnim2.DelayTime = TimeSpan.FromSeconds(.25f);
+
+                _PBGradientStop2.StartAnimation("Offset", _stop2offsetAnim2);
+
+                _pulseColor = _compositor.CreateColorKeyFrameAnimation();
+                _pulseColor.InsertKeyFrame(0, _transparent);
+                _pulseColor.InsertKeyFrame(1, _transparent);
+                _pulseColor.Duration = TimeSpan.FromSeconds(1.5);
+
+                _PBGradientStop1.StartAnimation(nameof(_PBGradientStop1.Color), _pulseColor);
+
+                // Create scale animation the helps with the final visual end result of create a pulsing effect
+                _scale = _compositor.CreateVector3KeyFrameAnimation();
+                _scale.InsertKeyFrame(0, new Vector3(0, 0, 0));
+                _scale.InsertKeyFrame(1, new Vector3(1, 1, 0));
+                _scale.Duration = TimeSpan.FromSeconds(1.5);
+                _scale.IterationCount = 50;
+
+                _pulseVisual.StartAnimation("Scale", _scale);
+
+                // Turn off the stopwatch and change text back to Start
+                _dt.Stop();
+                _stopwatch.Start();
+                Timer.Content = ("Start");
+
+                // Set our boolean flag for the stopwatch animation to false to indicate it's off
+                animationOn = false;
+            }
+        }
+
+        // Method that sets up the stopwatch to display the time or "Start" when clicked on and off
+        private void Dt_Tick(object sender, object e)
+        {
+            TimeSpan ts = _stopwatch.Elapsed;
+            _currentTime = String.Format("{0:00}:{1:00}:{2:00}", ts.Minutes, ts.Seconds, ts.Milliseconds / 10);
```

Comment:
I'd expect milliseconds would display as a decimal rather than with a colon. E.g. 1:23.5 rather than 1:23:05. Especially since you're defaulting to 2 digits for milliseconds which means it will flash between 2 digits and 3 digits depending on the current millisecond value.

Reply:
changed to this